server/ServerFSM.cpp

```diff
@@ -2447,7 +2447,14 @@ sc::result WaitingForTurnEnd::react(const TurnOrders& msg) {
     const PlayerConnectionPtr& sender = msg.m_player_connection;
 
     auto order_set = boost::make_unique<OrderSet>();
-    ExtractTurnOrdersMessageData(message, *order_set);
+    try {
+        ExtractTurnOrdersMessageData(message, *order_set);
+    } catch (const std::exception& err) {
+        // incorrect turn orders. disconnect player with wrong client.
+        sender->SendMessage(ErrorMessage(UserStringNop("ORDERS_FOR_WRONG_EMPIRE")));
```

Comment:
Fixed. But I've added it in new commit which shouldn't be cherry-picked because even if we get new point-release it will be possible to have old 0.4.8 client connects to 0.4.8.1 server.

Reply:
It sounds to me like you are saying you think the first commit would be reasonable to cherrypick for a bugfix release, but that the second commit would not.  But it seems to me they both should be cherrypicked. 

Could you explain more specifically how you see the possibility of a 0.4.8 client connecting to a 0.4.8.1 server impacts this at all, and especially why it would mean that the second commit not be cherry picked?